lib/symbol-store.js

```diff
@@ -49,6 +49,21 @@ export default class SymbolStore {
     let firstLetter = prefix[0].toLowerCase()
     let symbolsByWord = new Map()
     let wordOccurrences = new Map()
+    let seen = new Map()
```

Comment:
I think we should be able to use a `Set` here. Also, what about renaming the variable from `seen` to `builtinSymbolsByWord`?